core/src/main/scala/cats/Foldable.scala

```diff
@@ -251,7 +251,7 @@ import simulacrum.typeclass
   /**
    * Behaves like sequence_, but uses [[Unapply]] to find the
    * [[Applicative]] instance for `G` - used when `G` is a
-   * type constructor with two or more parameters such as [[scala.util.Either]]
+   * type constructor with two or more parameters such as `scala.util.Either`
```

Comment:
I totally endorse this. I may-or-may-not have made a snarky tweet about Scaladoc today related to this.